quickstarts/ansible-automation-controller/config.yml

```diff
@@ -0,0 +1,45 @@
+
+
+slug: ansible-automation-controller
+description: |
+  ## Why monitor Ansible Automation Controller?
+  An ideal Ansible Automation Controller performance monitor must offer comprehensive and actionable information that software developers need to troubleshoot an application successfully. Some key components are:
+
+  - Track the efficiency of business transactions starting using the Ansible Automation Controller server.
```

Comment:
```suggestion
  - Track the efficiency of your Ansible Automation Controller server's business transactions.
```

Reply:
Hi @**paperclypse** ,
Reviewed text correction has been changed. Kindly review it.